src/Discord.Net/Entities/Emotes/IEmote.cs

```diff
@@ -1,10 +1,6 @@
-using System;
-using System.Collections.Generic;
-using System.Text;
-
 namespace Discord
 {
-    public interface IEmote
+    public interface IEmote : IMentionable // TODO: Is `Mention` the correct verbage here?
```

Comment:
I'd say ITag instead of IMentionable; as it stays mostly in line with the current naming.